<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxConnexion">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection au serveur</string>
   </property>
   <widget class="QLabel" name="labelIPadresse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>36</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP du serveur :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Num√©ro de port du serveur :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtatConnexion">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInfoClient">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>621</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QPushButton" name="pushButtonNomOrdi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'ordinateur distant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNomUtilisateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de l'utilisateur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArchiProcesseur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Architecture processeur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOSordinateur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OS de l'ordinateur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomOrdi">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNomUtilisateur">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArchiProcesseur">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOSordinateur">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
